### PR TITLE
Fix/vl/fix transforms in formula recognition

### PR DESCRIPTION
**Pull request by morkovka1337**
1. Add missing parameter — probability to apply transforms. 
2. Add `RandomChoice` for bolding transforms as they should not be applied all (only one should be selected).
3. Fix transform variable in train. 
4. Also make transform in handwritten polynomials config consistent with OMZ one.

### CODE REVIEW

File: pytorch_toolkit/formula_recognition/im2latex/data/utils.py [hunk 1]
```diff
@@ -452,10 +452,17 @@ def collate_fn(sign2id, batch, *, batch_transform=None):
...
+    bolding_transforms = []
     if transforms_list:
         for transform in transforms_list:
             transform_name = transform.pop('name')
-            transforms.append(TRANSFORMS[transform_name](**transform))
+            transform_prob = transform.pop('prob', 1.0)
+            if transform_name == "TransformRandomBolding":
+                bolding_transforms.append(TRANSFORMS[transform_name](**transform))
+            else:
+                transforms.append(RandomApply([TRANSFORMS[transform_name](**transform)], p=transform_prob))
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/formula_recognition/im2latex/data/utils.py:463]: Can we use this call alwasy?

Comment (morkovka1337) [reply pytorch_toolkit/formula_recognition/im2latex/data/utils.py]: The logic was hidden into `RandomBoldingTransform` class

File: pytorch_toolkit/formula_recognition/im2latex/data/utils.py [hunk 2]
```diff
@@ -238,55 +238,65 @@ class TransformRandomBolding:
...
+                res[i] = cv.GaussianBlur(res[i].astype(np.uint8), (3, 3), self.sigmaX)
+                res[i] = (res[i] >= self.res_threshold).astype(np.uint8) * 255
+
+            return res
+
+    def __init__(self, params):
+        assert isinstance(params, (tuple, list))
+        self.list_of_transforms = []
+        for param in params:
+            self.list_of_transforms.append(TransformRandomBolding.SingleBoldingTransform(**param))
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/formula_recognition/im2latex/data/utils.py:290]: Why do you need to store all transforms if you constantly do one of them only?

Comment (Ilya-Krylov) [reply pytorch_toolkit/formula_recognition/im2latex/data/utils.py:290]: sry, my fauld

**Review by Ilya-Krylov — APPROVED** [review]

**Pull request merged**